src/engine-actions/get-network.js

```diff
@@ -0,0 +1,14 @@
+const { getInfo } = require('../lnd-actions')
+
+/**
+ * Returns the lnd network (e.g. mainnet, regtest, etc)
+ *
+ * @returns {Promise<string>} network
+ */
+async function getNetwork () {
+  const { chains } = await getInfo({ client: this.client })
+  const { network } = chains[0]
```

Comment:
What if there is more than one chain?

Reply:
This would break, however `lnd-engine` assumes one LND instance per chain so it seems like a safe assumption.

Do you think there's a better way to handle?

Reply:
Given that assumption, it sounds like there could still be multiple chains? If so, then the interface might have to take a parameter that uniquely specifys one chain.